boards/arm/96b_carbon/96b_carbon.dts

```diff
@@ -84,8 +84,8 @@
 	bt-hci@0 {
 		compatible = "zephyr,bt-hci-spi";
 		reg = <0>;
-		reset-gpios = <&gpiob 4 0>;
-		irq-gpios = <&gpiob 1 0>;
+		reset-gpios = <&gpiob 4 GPIO_ACTIVE_HIGH>;
```

Comment:
Since the driver is using logical levels to drive the reset signal I believe we should have here `GPIO_ACTIVE_LOW` as the reset is activated when the line is low. The driver code would need to be updated to set value `1` to activate the reset. Otherwise - this is an open point, haven't been discussed so far - we could also have the driver use raw / physical signals. In this case it should use `gpio_pin_set_raw` function to set the pin value and we shouldn't have here `GPIO_ACTIVE_HIGH` flag.